video/video.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/guardian/new-encodings-endpoints/common"
+	"log"
+)
+
+/*
+This script looks up a video in the interactivepublisher database and returns a URL, if it can be found, in a location header
+*/
+
+func HandleEvent(ctx context.Context, event *events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	config, configErr := common.NewConfig()
+	if configErr != nil {
+		log.Printf("ERROR Could not initialise config: %s", configErr)
+		return nil, configErr
+	}
+
+	ops := common.NewDynamoDbOps(config)
+
+	foundContent, errResponse := common.FindContent(ctx, &event.QueryStringParameters, ops, config)
+	if errResponse != nil {
+		switch errResponse.StatusCode {
+		case 404:
+			return common.MakeResponseRaw(404, aws.String("No content found.\n"), "text/plain"), nil
+		default:
+			return errResponse, nil
+		}
+	}
+
+	if _, ok := (event.QueryStringParameters)["poster"]; ok {
```

Comment:
I'd say `if _, havePoster` not `if _, ok` as the latter is harder to read

Reply:
Done.